alphadia/outputtransform.py

```diff
@@ -845,6 +847,8 @@ def _build_run_stat_df(
         Dataframe containing the statistics
 
     """
+    optimization_manager_path = os.path.join(folder, "optimization_manager.pkl")
```

Comment:
Can you use `OPTIMIZATION_MANAGER_PATH` here?

Reply:
Changed.